docs/api/selector.md

```diff
@@ -1,66 +1,71 @@
-# enzyme Selectors
+# Enzyme Selectors
+
+Many methods in Enzyme's API accept a *selector* as an argument. Selectors in Enzyme can fall into
+one of the following five categories:
 
-Many methods in enzyme's API accept a *selector* as an argument. Selectors in enzyme can fall into
-one of the following four categories:
 
 
 ### 1. A Valid CSS Selector
 
-enzyme supports a subset of valid CSS selectors to find nodes inside a render tree. Support is as
+Enzyme supports a subset of valid CSS selectors to find nodes inside a render tree. Support is as
 follows:
 
 - class syntax (`.foo`, `.foo-bar`, etc.)
-- tag syntax (`input`, `div`, `span`, etc.)
+- element syntax (`input`, `div`, `span`, etc.)
 - id syntax (`#foo`, `#foo-bar`, etc.)
-- prop syntax (`[htmlFor="foo"]`, `[bar]`, `[baz=1]`, etc.);
-
-**Note -- Prop selector**
-Strings, numeric literals and boolean property values are supported for prop syntax
-in combination of the expected string syntax. For example, the following
-is supported:
-
-```js
-const wrapper = mount((
-  <div>
-    <span foo={3} bar={false} title="baz" />
-  </div>
-));
+- attribute syntax (`[href="foo"]`, `[type="text"]`, etc.)
 
-wrapper.find('[foo=3]');
-wrapper.find('[bar=false]');
-wrapper.find('[title="baz"]');
-```
-
-Further, enzyme supports combining any of those supported syntaxes together to uniquely identify a
-single node.  For instance:
+Further, Enzyme supports combining any of those supported syntaxes together to uniquely identify a
+single node. For instance:
 
 ```css
 div.foo.bar
 input#input-name
-label[foo=true]
+a[href="foo"]
 ```
 
-enzyme also gives support for the following contextual selectors
+Enzyme also gives support for the following contextual selectors
 
-```
+```css
 .foo .bar
 .foo > .bar
 .foo + .bar
 .foo ~ .bar
 .foo input
 ```
 
-
 **Want more CSS support?**
 
 PR's implementing more support for CSS selectors will be accepted and is an area of development for
-enzyme that will likely be focused on in the future.
+Enzyme that will likely be focused on in the future.
+
 
 
+### 2. Prop Selector
 
-### 2. A React Component Constructor
+In addition to traditional CSS selectors, Enzyme supports using a React prop like an Attribute Selector as if it were an HTML attribute. Strings, Numbers, and Boolean property values are supported.
 
-enzyme allows you to find components based on their constructor. You can pass in the reference to
+```js
+const wrapper = mount((
+  <div>
+    <span foo={3} bar={false} title="baz" />
+  </div>
+));
+
+wrapper.find('[foo=3]');
+wrapper.find('[bar=false]');
+wrapper.find('[title="baz"]');
+```
+
+**The Key and Ref Prop**
+
+While in most cases, any React prop can be used, there are exceptions. The `key` and `ref` props will never work. This decision comes from how React uses these props internally and should not be relied upon.
```

Comment:
"internally, which means they should not be relied upon"

Reply:
updated!